### PR TITLE
Add a port setting to build script

### PR DESCRIPTION
**Pull request by polaroi8d**
JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

File: tools/build.py [hunk 1]
```diff
@@ -119,6 +120,7 @@ def generate_build_options(arguments):
     build_options.append('-DFEATURE_PROFILE=%s' % PROFILE)
 
     build_options.append('-DFEATURE_DEBUGGER=%s' % arguments.jerry_debugger)
+    build_options.append('-DJERRY_DEBUGGER_PORT=%s' % arguments.jerry_debugger_port)
```
Comment (robertsipka) [inline tools/build.py]: set FEATURE_DEBUGGER_PORT here.

File: jerry-core/CMakeLists.txt [hunk 2]
```diff
@@ -40,6 +41,7 @@ endif()
 # Status messages
 message(STATUS "FEATURE_CPOINTER_32_BIT   " ${FEATURE_CPOINTER_32_BIT})
 message(STATUS "FEATURE_DEBUGGER          " ${FEATURE_DEBUGGER})
+message(STATUS "FEATURE_DEBUGGER_PORT     " ${JERRY_DEBUGGER_PORT})
```
Comment (robertsipka) [inline jerry-core/CMakeLists.txt]: ${FEATURE_DEBUGGER_PORT}

File: jerry-core/CMakeLists.txt [hunk 3]
```diff
@@ -175,6 +177,9 @@ if(FEATURE_DEBUGGER)
   set(DEFINES_JERRY ${DEFINES_JERRY} JERRY_DEBUGGER)
 endif()
 
+# Debugger port
+set(DEFINES_JERRY ${DEFINES_JERRY} JERRY_DEBUGGER_PORT=${JERRY_DEBUGGER_PORT})
```
Comment (robertsipka) [inline jerry-core/CMakeLists.txt]: ${FEATURE_DEBUGGER_PORT}

**Comment by polaroi8d** [comment]
Thanks @robertsipka, updated the PR :+1:

File: tools/build.py [hunk 4]
```diff
@@ -119,6 +120,7 @@ def generate_build_options(arguments):
     build_options.append('-DFEATURE_PROFILE=%s' % PROFILE)
 
     build_options.append('-DFEATURE_DEBUGGER=%s' % arguments.jerry_debugger)
+    build_options.append('-DFEATURE_DEBUGGER_PORT=%s' % arguments.jerry_debugger_port)
```
Comment (robertsipka) [inline tools/build.py]: Use '%d' here instead '%s'.

**Comment by polaroi8d** [comment]
Thanks @robertsipka, updated the PR :+1:

**Comment by robertsipka** [comment]
LGTM (informally)

File: jerry-core/CMakeLists.txt [hunk 5]
```diff
@@ -19,6 +19,7 @@ project (${JERRY_CORE_NAME} C)
 # Optional features
 set(FEATURE_CPOINTER_32_BIT  OFF     CACHE BOOL   "Enable 32 bit compressed pointers?")
 set(FEATURE_DEBUGGER         OFF     CACHE BOOL   "Enable JerryScript debugger?")
+set(FEATURE_DEBUGGER_PORT    "5001"  CACHE STRING "Use default or other port number?")
```
Comment (zherczeg) [inline jerry-core/CMakeLists.txt]: Set debugger port number (default: 5001).

**Pull request merged**